docs/usage.rst

```diff
@@ -33,180 +6,152 @@ a SQLite_ database.
     import audeer
 
     _cwd_root = os.getcwd()
-    _tmp_root = audeer.mkdir(os.path.join('docs', 'tmp'))
+    _tmp_root = audeer.mkdir('docs', 'tmp-usage')
     os.chdir(_tmp_root)
 
 
-.. _file-system-example:
-
-File-system example
--------------------
+.. _usage:
 
-The class
-:class:`audbackend.backend.FileSystem`
-implements
-:class:`audbackend.backend.Base`
-for a standard file system.
+Usage
+=====
 
+With the help of :mod:`audbackend`
+a user can store files
+in a repository
+on a storage system.
 
-We start by registering the class
-(in fact,
-the class is registered
-by default,
-but it doesn't hurt
-if we do it again).
+The user can select between different :ref:`interfaces <interfaces>`
+that influence how the data is structured
+and presented to the user.
+In addition,
+:mod:`audbackend` supports different storage systems,
+so called :ref:`backends <backends>`.
+The names of all available backends can be listed by:
```

Comment:
```suggestion
The names of all available backends can be listed with:
```

Reply:
At least sounds better to me, but I am not 100% sure :)

Reply:
I also had first "with", but it sounded to German to me ;)

I searched a littel bit and "listed by" seems to be used to refer to persons oding the listing, whereas "listed with" refers to objects that are listed together with the object before. So maybe "by" is then better as the code that follows is responsible for the listing and is not listed together with the text before?

Reply:
Ok, let's stick with "by" then :)